### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only build validation fixes and does not change other content.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: @MonicaRush

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
 # Windows Console Documentation
-Welcome to the Windows Console documentation repo. The MSDN Console Docs are generated from the markdown stored in this repo.
-The published docs can be found at [https://docs.microsoft.com/windows/console/](https://docs.microsoft.com/windows/console/)
+
+Welcome to the Windows Console documentation repo. The published articles are generated from the markdown stored in this repo and published at [Windows Console documentation](https://docs.microsoft.com/windows/console/).
```
Comment (jmart1428) [inline README.md:3]: "https://docs.microsoft.com/windows/console/" could be changed to a relative link.

**Comment by jmart1428** [comment]
#docutune-wait

File: README.md (hunk 1 above)
Comment (alexbuckgit) [reply README.md:3]: Files at the root of a repo are not published and require an absolute link when viewed in GitHub.

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@miniksa,DHowett

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Review by miniksa — APPROVED** [review]

**Pull request merged**